tests/lib/output.js

```diff
@@ -0,0 +1,16 @@
+import { resolve } from 'path';
+import rimraf from 'rimraf';
+import promisify from 'es6-promisify';
+import mkdirp from 'mkdirp';
+
+const rm = promisify(rimraf);
+
+export const outputPath = resolve(__dirname, '../output');
+
+export const setup = async () => {
```

Comment:
Both of these can just be:

```js
export const name = () => helper(foo);
```

since they are returning Promises & will be/are `await`ed in the tests.